SwiftTweaks/TweakTableCell.swift

```diff
@@ -180,13 +180,11 @@ internal final class TweakTableCell: UITableViewCell {
 			accessory.bounds = textField.bounds
 
 		case .stringList:
-			let textFieldFrame = CGRect(
-				origin: .zero,
-				size: CGSize(
-					width: bounds.width * TweakTableCell.stringTextWidthFraction,
-					height: bounds.height
-				)
-			).integral
+			let textFieldSize = self.textField.sizeThatFits(CGSize(
+				width: CGFloat.greatestFiniteMagnitude,
+				height: bounds.height
+			))
+			let textFieldFrame = CGRect(origin: .zero, size: CGSize(width: textFieldSize.width + 10.0, height: bounds.height))
```

Comment:
Why is there a `10.0` here? What does that mean, semantically? (As much as possible, we prefer to take "magic numbers" like this and give them a semantic name, eg `TweakTableCell.horizontalPadding`, so future-contributors can readily understand what they're doing.)